helm/vernemq/values.yaml

```diff
@@ -129,6 +129,11 @@ statefulset:
     timeoutSeconds: 5
     successThreshold: 1
     failureThreshold: 3
+  podAnnotations: {}
+#    prometheus.io/scrape: "true"
+#    prometheus/io/port: "8888"
```

Comment:
Not knowing anything about how this works, but is it correct that in one line a `.` is used and in the other a`/` is used:`prometheus.io` vs `prometheus/io` ?

Reply:
yes you're right there is a typo. I will fix it

Reply:
Wow, you're fast! :)